addok/shell.py

```diff
@@ -56,7 +56,7 @@ def save_history(self):
 
     @property
     def history_file(self):
-        return str(Path.home() / self.HISTORY_FILE)
+        return str(os.path.join(os.path.expanduser('~'),self.HISTORY_FILE))
```

Comment:
The `str()` conversion is not anymore required. Space after comma [says PEP8](https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements).